docker/Dockerfile

```diff
@@ -21,25 +21,22 @@ RUN BUILD_DEPS="curl ca-certificates xz-utils" && \
     apt-get -qq clean && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-COPY postgrest.conf /etc/postgrest.conf
-
+COPY start .
```

Comment:
Usually this file is called `entrypoint.sh` in Dockerfiles, how about if you change it and then have this line as:
```Dockerfile
COPY entrypoint.sh /entrypoint.sh
```